### PR TITLE
Dom/lookup docs

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: site/docs/transform/lookup.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  ],
+  ...
+}
+```
+
+For each data object in the main data source, the transform tries to find a matching objects in the secondary data source. An object matches if the value in the field specified by `lookup` is the same as the field specified in the `from.key`.
+
+{% include table.html props="lookup,from,as,default" source="LookupTransform" %}
+
+The secondary data reference (set with `from`) is an object that specifies how the lookup key should be matched to a second data source and what fields should be added.
```
Comment (kanitw) [inline site/docs/transform/lookup.md]: Missing an anchor (and link from "LookUpData" in the table).

**Comment by domoritz** [comment]
Done

**Comment by domoritz** [comment]
Fixed the build failure.

**Comment by domoritz** [comment]
Rebased

**Pull request merged**